kedro-org/kedro-viz
# Fix broken URL when active pipeline name changes on initial load

## Description

Resolves [#1879](https://github.com/kedro-org/kedro-viz/issues/1879)

![pid_fix](https://github.com/kedro-org/kedro-viz/assets/87735534/87340170-0efc-4d8e-bf53-fa55dc3d4f4b)

## Development notes

* Remove params check to update activePipeline

## QA notes

* Run `kedro viz run` -> Change the pipeline id in pipeline_registry of your kedro project -> Run `kedro viz run`. The URL should point to the new pipeline id.
* All tests should pass

## Checklist
